nodes/alarm_server.py

```diff
@@ -97,7 +101,7 @@ def update(self, srv):
         self.stamp = rospy.Time.now()
         
         node_name = "unknown" if srv.node_name is "" else srv.node_name
-        parameters = self.parse_json_str(srv.parameters)
+        parameters = '' if srv.parameters is '' else json.loads(srv.parameters)
```

Comment:
This change causes the cpp tests to fail. If I try to raise an alarm with an invalid JSON string, there will be a crash. I don't mind the different approach you're taking as long as it can handle some acceptable bad input gracefully.

As a reminder try to always run `rostest ros_alarms ros_alarms_integration.test` before PR'ing.

Tests FTW!!